scripts/cm_syslog.py

```diff
@@ -16,8 +16,8 @@
 
 if __name__ == '__main__':
     parser = mc.get_mc_argument_parser()
-    parser.add_argument('-k', '--keyword', help="Keyword for comments [defaults = write: 'general', --view: 'all']", default=None)
-    parser.add_argument('-c', '--comment', help="Comment on part", default=None)
+    parser.add_argument('-k', '--keyword', help="Keyword for comments [defaults: write-->'general', --view-->'all']", default=None)
+    parser.add_argument('-c', '--comment', help="Test for log entry.", default=None)
```

Comment:
Is this supposed to be "test" and not, say, "text"?

Reply:
embarrassingly yes